responses/11_wrong-commit-id.md

```diff
@@ -1,43 +1,6 @@
 Good guess, but the commit SHA ID that added the `.env` file is 848cd8c2043f6161a4f0043bffee212777281494
 
-We'll need to remove the contents of this commit, as well as the commit that removed it from the history.
-
-## Step 13: Remove historical reference to a previous `.env` file
-
-We can do this with the following commands:
-
-1. Since we cloned the repository earlier, let's run `git checkout master` to put us back on the master branch
-1. Run `git pull` to update your local repository with the changes we merged from the contributor's pull request
-1. Run `git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch .env" HEAD` to remove the historical reference to the `.env` file
```

Comment:
@a-a-ron this command didn't get rid of the commits from my history 😢

Reply:
@hectorsector this command doesn't remove the commit itself from the history but rather remove all content within the commit relating to the `.env` file